packages/vue/src/auth/mapAuthState.ts

```diff
@@ -217,7 +239,9 @@ export const useFronteggAuthGuard = (options?: FronteggAuthGuardOptions) => {
         if (redirectUrl) {
           localStorage.setItem(FRONTEGG_AFTER_AUTH_REDIRECT_URL, redirectUrl);
         }
-        fronteggAuth.loginActions.requestHostedLoginAuthorize();
+
+        // first time is not a good naming. Here we pass true to check if the active url is oauth/callback to avoid silent refresh and fix a step up bug related to Fallback route
+        fronteggAuth.loginActions.requestHostedLoginAuthorizeV2({firstTime: true, shouldRedirectToLogin: true});
```

Comment:
We should test this fix carefully because it will load all the initial app requests again

Reply:
Also `shouldRedirectToLogin: true` can introduce breaking change?